db/migrate/20220402140925_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
+      t.string :posts_counter
+
```

Comment:
- [ ] I would recommend fixing the data type for the `bio` and `posts_counter` column.

```ruby
create_table :users do |t|
    ...
    t.text :bio
    t.integer :posts_counter
    ...
end
```